presto-docs/src/main/sphinx/security/tls.rst

```diff
@@ -27,7 +27,7 @@ that confirms the information is correct:
     Enter keystore password:
     Re-enter new password:
     What is your first and last name?
-      [Unknown]:  presto-coordinator
+      [Unknown]:  presto-coordinator-hostname.com
```

Comment:
Let's get rid of `.com`, since this is a valid domain name (that might belong to someone else). If we want to have a TLD, `.example` from [RFC 2606](https://tools.ietf.org/html/rfc2606) is appropriate here.